crates/holochain/tests/sharded_gossip/mod.rs

```diff
@@ -414,7 +430,13 @@ async fn test_zero_arc_no_gossip_4way() {
                         c.call::<_, Option<Record>>(&zome, "read", hash.clone())
                             .await
                             .is_some(),
-                        |x: &bool| *x,
+                        |x: &bool| {
```

Comment:
Faulty logic, the check would pass even when the condition wasn't met. This check now matches the assertion that will be run so if the test needs to wait longer than 1 tick it will